File: API_FLUTTER/ApiFlutter/ApiFlutter/Models/Vehiculo.cs
using System.ComponentModel.DataAnnotations;

namespace ApiFlutter.Models
{
    public class Vehiculo
    {
        [Key]
        public string BusSerie { get; set; }
        [Required]
        public string placa {  get; set; }
        [Required]
        public TimeSpan hora { get; set; }
        [Required]
        public string direccion { get; set; }
        [Required]
        public string LugarLlegada { get; set; }
        [Required]
        public string LugarSalida { get; set; }
    }
}

File: lib/Vehiculo.dart
class Vehiculo {
  final String BusSerie;
  final String placa;
  final String hora;
  final String direccion;
  final String LugarLlegada;
  final String LugarSalida;

  const Vehiculo(
      {required this.BusSerie,
      required this.placa,
      required this.hora,
      required this.direccion,
      required this.LugarLlegada,
      required this.LugarSalida});

  String getbusSerie() {
    return BusSerie;
  }

  String getPlaca() {
    return placa;
  }

  String getHora() {
    return hora;
  }

  String getDireccion() {
    return direccion;
  }

  String getLugarLL() {
    return LugarLlegada;
  }

  String getLugarS() {
    return LugarSalida;
  }

  factory Vehiculo.fromJson(Map<String, dynamic> json) => Vehiculo(
      BusSerie: json['busSerie'],
      placa: json['placa'],
      hora: json['hora'],
      direccion: json['direccion'],
      LugarLlegada: json['lugarLlegada'],
      LugarSalida: json['lugarSalida']);
}
